src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs

```diff
@@ -88,16 +88,16 @@ public async Task<int> GetCountByOrganizationAsync(Guid organizationId, string e
         }
     }
 
-    public async Task<int> GetOccupiedSeatCountByOrganizationIdAsync(Guid organizationId)
+    public async Task<OrganizationSeatCounts> GetOccupiedSeatCountByOrganizationIdAsync(Guid organizationId)
     {
         using (var connection = new SqlConnection(ConnectionString))
         {
-            var result = await connection.ExecuteScalarAsync<int>(
+            var result = await connection.QueryFirstOrDefaultAsync<OrganizationSeatCounts>(
```

Comment:
This should always return a single object, I think `QueryAsync` is what you want here.